### PR TITLE
 Practice 2 - JavaScript Basics. Part 2.

### PR DESCRIPTION
**Pull request by zviktorya**
Task 1: Write a function which takes either a number or a string.
Task 2: Write a function which will run through the array and print each element of the array to the console and total number of elements.
Task 3: Write a function that will count and output to the console the number of even and odd elements in the array.
Task 4: Write a function that checks if the same elements in the array.
Task 5: Write a function that returns the maximum value in an array.
Task 6: Write a function that determines whether a number is prime.

### CODE REVIEW

File: src/ex2_js-basics-part2/task-01.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function checkNumbOrStr(arg) {
```
Comment (aneelia) [inline src/ex2_js-basics-part2/task-01.js]: checkType

File: src/ex2_js-basics-part2/task-02.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+function outputOfArray(array) {
```
Comment (aneelia) [inline src/ex2_js-basics-part2/task-02.js]: printElements

File: src/ex2_js-basics-part2/task-03.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+function getCountElement(array) {
+    let sumEven = 0;
+    let sumZero = 0;
```
Comment (aneelia) [inline src/ex2_js-basics-part2/task-03.js]: один let

File: src/ex2_js-basics-part2/task-01.js (hunk 1 above)
Comment (zviktorya) [reply src/ex2_js-basics-part2/task-01.js]: Done

File: src/ex2_js-basics-part2/task-02.js (hunk 2 above)
Comment (zviktorya) [reply src/ex2_js-basics-part2/task-02.js]: Done

File: src/ex2_js-basics-part2/task-03.js (hunk 3 above)
Comment (zviktorya) [reply src/ex2_js-basics-part2/task-03.js]: Done

**Pull request merged**